README.md

```diff
@@ -74,6 +79,11 @@ Follow the instructions below.
 ### Environment set up
 Build a docker image with Dockerfile and create a docker container.
 
+```
+docker build -t bert-ja .
+docker run -v `pwd`:/work --rm  -it bert-ja bash
```

Comment:
I've successfully run the script `src/data-download-and-extract.py` inside a docker container created this way.